python/test_python.py

```diff
@@ -82,33 +82,41 @@ def test_read_file(self, docker_container):
 
 
 class TestArgs:
+    """Test that args can be passed to script"""
     @pytest.mark.parametrize(
         'docker_container',
         ['python arguments.py "Argument Number 1"'],
         indirect=True,
     )
     def test_args(self, docker_container):
         docker_container.wait()
-        assert docker_container.logs() == b'Argument Number 1\n'
+        assert docker_container.logs() == b'argument number 1\n'
 
 
 class TestReadJsonFile:
+    """Test that a JSON file is read correctly"""
     @pytest.mark.parametrize(
         'docker_container',
-        ['python read_json.py < person-records.json'],
+        ['python read_json.py < person-records.json'], # < person-records.json isn't doing anything here since the filepath is in the script
```

Comment:
@nicholaides I realized that this test is weird. Is it ok just have the filepath given in the `read_json.py` script itself? or does it need to be passed in as an argv?

Reply:
Assuming this test is for #6, this is good-- I specifically want it to be reading from stdin.

Reply:
Sorry, I misunderstood. Yes, the read_json.py should be reading from stdin, similar to the `stdin.py` script-- it shouldn't know there's a file named `person-records.json`.

Reply:
I updated the script and test. Lemme know if it looks good